test/liveValidation/payloads/dateTime.json

```diff
@@ -74,7 +74,7 @@
         "validationEnvironment": false,
         "ring": null,
         "registrationInfo": {
-          "expirationTime": "2024-02-29T02:33:35",
+          "expirationTime": "2024-02-19T02:33:35",
```

Comment:
02-29 is for negative test and 02-19 is for positive test. Can you keep them both?